content/en/cloudcraft/getting-started/connect-amazon-eks-cluster-with-cloudcraft.md

```diff
@@ -0,0 +1,105 @@
+  <p>The ability to scan Amazon EKS clusters and AWS accounts is only available to Cloudcraft Pro subscribers. Check out <a href="https://www.cloudcraft.co/pricing">our pricing page</a> for more information.</p>
+</section>
+
+## Prerequisites
+
+Before connecting your Amazon EKS clusters with Cloudcraft, you need to connect your AWS account and generate diagrams that include your clusters.
+
+Linked below, you will find documentation to help you connect your AWS account and familiarize yourself with Cloudcraft.
+
+- [Connect your AWS account with Cloudcraft][4]
+- [Create your first live AWS diagram][5]
+
+<p>You will also want to <a referrerpolicy="no-referrer" rel="noreferrer external help" href="https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html">install and configure <code>kubectl</code></a>, a tool that lets you control Kubernetes clusters through the command line. AWS requires version 1.21 or higher, but we recommend using the latest version to avoid issues.</p>
+
+## Authorizing the Cloudcraft IAM role for view-only
+
+Start by opening a blueprint with an existing Amazon EKS cluster or using the **Auto Layout** feature to generate a new blueprint.
+
+With your AWS environment mapped into a blueprint, select the Amazon EKS cluster that you wish to scan, and click the **Enable cluster scanning** button that appears in the component toolbar.
+
+{{< img src="cloudcraft/getting-started/connect-amazon-eks-cluster-with-cloudcraft/enable-cluster-scanning.png" alt="Interactive Cloudcraft diagram showing an AWS EKS cluster with enable cluster scanning button highlighted." responsive="true" style="width:100%;">}}
+
+The next screen will provide step-by-step instructions, so open a terminal window and proceed.
+
+As the Amazon EKS cluster creator or user with admin access, open the aws-auth ConfigMap file with `kubectl`.
+
+```
+kubectl edit -n kube-system configmap/aws-auth
+```
+
+With the `aws-auth.yaml` file open in your favorite text editor, add the role details to the *mapRoles* section of the file, just after under the *data* section.
+
+```
+data:
+  mapRoles: |
+    - rolearn: <arn-for-the-readonly-cloudcraft-iam-role>
+      groups:
+        - cloudcraft-view-only
+```
+
+If the section does not exist, add it. Once done, save the file and exit.
+
+<section class="alert alert-info">
+  <p><em>groups</em> refer to groups within your cluster to which you mapped the role to. The <a referrerpolicy="no-referrer" rel="noreferrer external help" href="https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings">Default Roles and Role Bindings</a> page in the Kubernetes documentation website should give you more information.</p>
+</section>
+
+<section class="alert alert-danger">
+  <p>Typos and syntax errors can affect the permissions of all IAM users and roles updated within the ConfigMap file. We recommend adding a YAML linter to your favorite text editor to prevent issues.</p>
+</section>
```

Comment:
```suggestion
<div class="alert alert-info">`groups` refer to groups in your cluster to which the role is mapped. For more information, see [Default Roles and Role Bindings][8] in the Kubernetes documentation.
</div>

<div class="alert alert-danger">Typos and syntax errors can affect the permissions of all IAM users and roles updated in the ConfigMap file. To prevent this from occuring, Cloudcraft recommends adding a YAML linter to your text editor.
</div>
```